src/components/PriceSummary.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+export function PriceSummary({ products }){
+    console.log(products)
```

Comment:
Take care of cleaning up consolelogs

Reply:
Yes will take care of this. Thanks for poinitng out.